Contribute/index.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Microsoft Docs contributor guide overview
-description: The guide describes how you can contribute to the Microsoft documentation site docs.microsoft.com.
+title: Contributor guide overview
```

Comment:
This impacts SEO. @jehchow do you have an opinion on the public contributor guide page?

Reply:
Reverted